presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2342,4 +2343,17 @@ public FeaturesConfig setPrefilterForGroupbyLimit(boolean prefilterForGroupbyLim
         this.prefilterForGroupbyLimit = prefilterForGroupbyLimit;
         return this;
     }
+
+    public boolean isOptimizeJoinProbeForEmptyBuildRuntimeEnabled()
+    {
+        return isOptimizeJoinProbeWithEmptyBuildRuntime;
+    }
+
+    @Config("join.optimize-probe-for-empty-build-runtime")
```

Comment:
Lets start this config as well with optimizer?

Reply:
I am not sure. As this is not an optimizer field like other optimizers which run at compile time for plan generation. It's an optimization at runtime.

Reply:
Sure, will update since both of you think this is a better naming.